src/format.ts

```diff
@@ -145,6 +167,36 @@ const builtinFormats = [
     // todo: potential opt: markdown flavor
     // todo: potential opt: window header level (h1, h2, etc)
   },
+  {
+    id: 'markdownList',
+    label: () => 'Markdown list',
+    transforms: () => ({
+      // todo: make special char escaping more robust across popular markdown renderers (github, etc); https://stackoverflow.com/questions/13824669/how-do-you-write-a-link-containing-a-closing-bracket-in-markdown-syntax
+      text: {
+        windowStart: ({ seq }) => `## ${getNumberedWindowText(seq)}\n\n`,
+
+        tab: ({ tab: { title, url } }) =>
+          `- [${
+            // wrap
+            (`${title || url!} | ${getDomain(url)}`) // wrap
```

Comment:
I've included the domain in the link title, as it's easier to quickly grok what type of link each one is just by looking at the list